src/uniref/components/entry/EntryMain.tsx

```diff
@@ -23,4 +21,4 @@ const EntryMain = ({ transformedData }: EntryMainProps) => (
   </>
 );
 
-export default memo(EntryMain, isSameEntry);
+export default memo(EntryMain);
```

Comment:
I seem to remember this was important incase one entry linked to another, then memo wouldn't know what has changed. Is this no longer an issue?

Reply:
Regarding this if that was an issue in the past I believe it is not anymore, I've tested multiple links through entries and I haven't been able to reproduce the issue.
One thing I could think it would have been useful is if the object passed to the EntryMain component is new, but because it's been created through the data transformers after a re-render, and we would want to detect that the data it refers to is the same by checking if it's the same accession. It feels a bit hacky, and I've checked rerenders without this and it has been nothing crazy.
In this case this bit of code was blocking the needed rerender when we know that there are variants or not, because that's done asynchronously after the first render, so I think it's worth removing it.